### DISCUSSION TITLE
Is there some desktop app to open .map files?

### DISCUSSION THREAD

**Discussion by Beeperz** [mapsforge/mapsforge#1499] [Q&A] [answered]
I have downloaded .map and .rtd files (packed in zip) from bicycle navigator site, to check is it’s maps suitable to me, before buying. But can’t open these files. Tried some android apps and Cruiser (gives error - not supported version). Any help? Files from here https://global.igpsport.com/map/mapinfo?devicename=iGS800

In text editor these files have this header: mapsforge binary OSM рЂ  Џж ЊТЁZ-уctФ{tЋ9Ч`· Mercatoremapsforge-map-writer-0.16…

![image](https://github.com/mapsforge/mapsforge/assets/5863515/72c62e1c-17b3-4e92-946c-3f4bd75a21c8)

**Comment by devemux86** [comment, marked as answer]
@Beeperz 

These map files mention that they are Mapsforge map files,
but perhaps they don't follow the Mapsforge map file format specification and have modified (?) it.

So they cannot be read by the official Mapsforge map readers, like Cruiser.

There are many free Mapsforge map providers, see the list here:
- https://github.com/mapsforge/mapsforge/blob/master/docs/Mapsforge-Maps.md

**Reply by Beeperz** [reply]
Thanx for providers list but I need to open these files.

**Comment by thiyakuso4** [comment]
[Cartograph maps](https://www.cartograph.eu/v3/) has both mobile and desktop version, that opens Mapsforge map files.

**Reply by Beeperz** [reply]
Get error while opening :(

**Comment by CartographMaps** [comment]
Your map file is a modified Mapsforge format (proprietary format by IGP) which does not work with the official Mapsforge library. I contacted IGP a while ago but they did not provide details about their modifications.

**Reply by oxenit** [reply]
@CartographMaps I successfully loaded an IGP map in your software 👌 Did you have new infos about their implementation ? I would like to update their map which is quite old for my GPS device.

**Reply by Beeperz** [reply]
How you did that? I tried now and still error.

**Reply by oxenit** [reply]
Perhaps my map is not exactly the same ? I have a iGS630 and I used the map "europe / France / Ile de France" if you wanna try. 
Importing in Cartograph 3.7.4 worked directly.

**Reply by Beeperz** [reply]
What option you use? Import map or some "Pro" variants? I nave not Pro.

**Reply by CartographMaps** [reply]
@oxenit Can you send me a download link to the map which is working for you? You can use the email address shown on "https://www.cartograph.eu/v3/contact/" if you want to send a private link.

**Reply by Beeperz** [reply]
https://qw20191008.oss-us-west-1.aliyuncs.com/Mapinfo/V3/RU01.zip

**Reply by fernandoglatz** [reply]
Look in my repository https://github.com/fernandoglatz/igpsport-map-updater. I can generate new, updated maps with more detail than IGPSPORT's own maps.

**Comment by CartographMaps** [comment]
@Beeperz Thanks, but I think you already sent me the map (which is not working). I am interested in @oxenit 's map.

**Reply by oxenit** [reply]
Here it is https://files.catbox.moe/hb9vzp.map

**Comment by mroeller** [comment]
@Beeperz I found a solution to open the new igpsport file format in cruiser:
open the .map file in a hex-editor and go to byte 24 (18h) this should be "80" in the ipgsport-file. change this to "00" and save the file. It now opens in cruiser, but there seem to be something else that needs to be changed. the tiles have large spacing in between.
